### ISSUE TITLE
Adding of actual pneumatic cannon

### ISSUE THREAD

**Issue by B1gb3ast** [yogstation13/yogstation-classic#289]
We have one now, but that's only the improvised version.

Suggestion is adding a compartmentalizer to research which you can use to tablecraft a legitimate pneumatic cannon with twice the capacity of the ghetto one.

Would need spriting of compartmentalizer to be done. Also a bluespace compartmentalizer. If there's anyone who feels creative, make two cylinder thingies. What the fuck is a compartmentalizer? Eh, it has some kind of pressure gauge on it. maybe. seems legit. Make something pretty.
Also, a bluespace pneumatic cannon with infinite space. TFW the research director fires you and he shoots you with twelve desks.

Required:
-Approval of you _lovely people_
-sprites for the compartmentalizer, bluespace compartmentalizer and bluespace pneumatic cannon.
-Someone to code it, which I'm currently doing because I'm bored.

**Comment by ShadowDeath6** [comment]
Tl;dr buffing an instantkill weapon

**Comment by B1gb3ast** [comment]
Adding more pneumatics, not buffing the ghetto one.

**Comment by Kmc2000** [comment]
Sprites sprited, up to b1gm3m3 now
